.gitignore

```diff
@@ -9,6 +9,7 @@ target/
 **/*.DS_Store
 *.DS_Store
 .DS_Store
+._*
```

Comment:
Please remove all those files from the pull-request. Adding them to .gitignore is not a solution.

Then please remove this line as well.